code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -77,10 +81,15 @@
 	name = "clown shoes"
 	icon_state = "clown"
 	item_state = "clown_shoes"
-	slowdown = SHOES_SLOWDOWN+1
-	var/footstep = 1	//used for squeeks whilst walking
```

Comment:
I know you're replacing what this was for but was this not used at all? Where's the code removal of whatever is using this?

Reply:
I couldn't find any reference to it anywhere.